mobify/progressive-web-scaffold
# Rename the build NPM script to dev:build

We can't have a `npm build` script, this is a built-in script that NPM runs for `npm install` and `npm link` as described at https://docs.npmjs.com/cli/build.

## Changes
- Renamed `build` to `dev:build`

## How to test-drive this PR
- Remove `node_modules`
- `npm i`
- Make sure webpack doesn't do any funny business (this used to happen in the `build` script)
- Clone the scaffold into a new folder and verify the same as above
